lib/modules/datasource/endoflife-date/readme.md

```diff
@@ -0,0 +1,32 @@
+[endoflife.date](https://endoflife.date) provides version and end-of-life information for different packages.
+
+To find the appropriate "package" name for the software you're trying to update, use the endoflife.date "All packages" API endpoint. You can find it in [the API documentation](https://endoflife.date/docs/api).
+
+This datasource uses `loose` versioning by default. If the software you are upgrading supports a versioning that is more strict, e.g. `semver`, it is recommended to specify that explicitly.
+
+**Usage Example**
+
+Imagine using Amazon EKS and wanting to update the versions in a terraform `.tfvars` file.
+
+An example `.tfvars` file would look as follows:
+
+```hcl
+# renovate: datasource=endoflife-date depName=amazon-eks versioning=loose
+kubernetes_version = "1.26"
+```
+
+then, add the following configuration to your `renovate.json`:
+
+```json
+{
+  "regexManagers": [
```

Comment:
Once the config is correct, do we want to add it to the [Preset Regex Managers](https://docs.renovatebot.com/presets-regexManagers/) page?

Code for the presets lives here:

- https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/regex-managers.ts
- https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/regex-managers.spec.ts

Reply:
Having this config in the presets makes sense, yes. It doesn't have anything to do with this PR per se, so I think it makes sense to add it in a separate PR where it can be discussed independently. 
I can take care of that PR, too.

What do you think?